master/pkg/model/experiment.go

```diff
@@ -423,7 +428,37 @@ func (t TrialLog) Proto() *apiv1.TrialLogsResponse {
 	default:
 		panic("log had no valid ID")
 	}
-	return resp
+
+	if t.Timestamp != nil {
+		tsProto, err := ptypes.TimestampProto(*t.Timestamp)
+		if err != nil {
+			return nil, errors.Wrap(err, "failed to convert log timestamp to proto")
+		}
+		resp.Timestamp = tsProto
+	}
+
+	if t.Level != nil {
```

Comment:
Maybe switch the cases; I have vague preferences for putting much shorter blocks first and avoiding negations in conditions, both of which are minor but also line up here with the general Go convention of putting error-like cases first.